public/css/style.css

```diff
@@ -28,6 +28,13 @@
     height: 150px;
 }
 
+.selected-image-display {
+    margin: 1px 1px -3px 1px;
+    padding: 0 0 0 0;
+    width: 300px;
+    height: 300px;
+}
+
```

Comment:
And actually, maybe we want to use horizontal percentages instead of pixel values? This way we won't have issues with the divs stacking on top of each other when the window is resized... Thoughts?